linera-chain/src/chain.rs

```diff
@@ -239,6 +240,8 @@ where
     pub unskippable_bundles: QueueView<C, TimestampedBundleInInbox>,
     /// Unskippable bundles that have been removed but are still in the queue.
     pub removed_unskippable_bundles: SetView<C, BundleInInbox>,
+    /// The heights of previous blocks that sent messages to the same recipients.
+    pub previous_message_blocks: MapView<C, ChainId, BlockHeight>,
```

Comment:
It's persisted, right? and restored on process startup?

Reply:
Yes, it's part of the chain state, which is persisted whenever we process a confirmed block.